### ISSUE TITLE
Customizer > Colors & Backgrounds: individual colors options can't be selected when using a screen reader software

### ISSUE THREAD

**Issue by liviopv** [Automattic/wp-calypso#55680]
### Steps to Reproduce

1. Select a theme that has color options available Customize > Colors & Backgrounds
2. Using a screen reader software (like Mac OS' Voice Over), navigate using Tab to My Sites > Appearance > Customize > Colors & Backgrounds
3. Navigate using Tab through the options in the screen
4. The individual color options (the circles) will not be found by the screen reader software  

### What I expected to happen

Users who rely on a screen reader software should be able to change individual colors in their theme

### What actually happened

The color circles are not found by the screen reading software. Voice Over can't even detect the predefined color palettes. [Jaws](https://www.freedomscientific.com/products/software/jaws/) is able to detect the palettes section (and it readers several different color codes, which can be confusing), but skips past the circles.

### Context

Customer report. The user gently shared a video of them using Jaws to navigate the interface, which can be found on 4192398-zen.

I was able to reproduce using Mac's Voice Over.

### Operating System

macOS

### OS Version

11.51.1

### Browser

Chrome/Chromium

### Browser Version(s)

92

### Is this specific to applied theme? If so, what is the theme name?

Themes with individual color options on Customize > Colors & Backgrounds

### Simple/Atomic

Simple, Atomic

### Number of Users Impacted

Some users (<50%)

### Reproducibility

Consistent

**Comment by jordesign** [comment]
Tested and confirmed in triage - it isn't possible to tab-select the color pallet circles.

**Comment by dpasque** [comment]
Tagging @Automattic/flow-patrol-create based on issue content

**Comment by yansern** [comment]
Add `role="button" tabindex="0"` to `#color-grid > li`.
The file to modify is `wp-content/mu-plugins/colors/colors.php` on WPCOM.

![image](https://user-images.githubusercontent.com/1287077/132483832-92f1ddc8-03ce-41e7-8868-0b665b5ebee1.png)

**Comment by simison** [comment]
This might be better fit on themes team since they've been maintaining the colors plugin, @jeffikus could your team have a look?

**Comment by yansern** [comment]
> This night be better fit on themes team since they've been maintaining the colors plugin, @jeffikus could your team have a look?

Oh thanks, now I know who to ping.


Phab patch has arrived though. D66676-code


https://user-images.githubusercontent.com/1287077/132870372-6f2090c0-32f0-47a1-bc78-a65cdfc83f99.mp4

**Comment by yansern** [comment]
Latest updates fixes also a11y issues on the color picker tiles below. Tabbing and pressing enter/space now works.

https://user-images.githubusercontent.com/1287077/133058616-166ac720-6ed3-4157-a004-f10659c3d0e6.mp4

**Comment by jerrysarcastic** [comment]
Is there a possible regression here?  I noticed that a few themes I visit today have odd grey swatches (that cannot be selected) and I am wondering are these normally hidden?

Some examples pictured here (Twenty Seventeen and Dara) but seems to affect any legacy theme that offers custom colors.  

 
<img width="481" alt="Screenshot on 2021-09-28 at 13-44-36" src="https://user-images.githubusercontent.com/1967448/135164602-85eb8909-3be7-4187-aa94-ce502adc0377.png">

<img width="434" alt="Screenshot on 2021-09-28 at 13-44-57" src="https://user-images.githubusercontent.com/1967448/135164612-cc6528d4-7fba-474b-ad18-ccebb1f653e5.png">

**Comment by simison** [comment]
@jeffikus ☝️ can you have a look?

**Comment by jeffikus** [comment]
@simison I've added it to our board for the team to take a look at while I'm AFK next week.

**Comment by alshakero** [comment]
I'll take a look since I'm probably the one who introduced the regression.

**Comment by alshakero** [comment]
Fix ready for review in D67925-code

**Comment by alshakero** [comment]
That's a lot for the elaborate report @jerrysarcastic and @yansern for the review! This is now fixed and deployed to production.
